exercicios/para-casa/nathalia-pereira.py

```diff
@@ -0,0 +1,62 @@
+# ------------------------------------------------PASSO 1------------------------------------------------
+# Use o arquivo `mais_ouvidas_2024.csv` para análise. Lembre-se de garantir que o carregamento foi feito com sucesso.
+#--------------------------------------------------------------------------------------------------------
+ 
+import pandas as pd
+
+df_mais_ouvidas = pd.read_csv('../../material/mais_ouvidas_2024.csv')
+
+print(df_mais_ouvidas.head())
+
+# ------------------------------------------------PASSO 2------------------------------------------------
+# Identifique as colunas que contêm números, como 'Spotify Streams', 'YouTube Views', etc., e converta essas colunas para o tipo numérico se estiverem em outro formato. (Use replace() e astype())
+#--------------------------------------------------------------------------------------------------------
+
+# print(df_mais_ouvidas.columns) # lista as colunas da tabela
+# print(df_mais_ouvidas.dtypes) # retorna os tipos de cada coluna da tabela
+print(df_mais_ouvidas.info()) # traz informações de colunas (qtde de linhas - nulos - tipos)
+print(df_mais_ouvidas.isnull().sum()) #traz a qtde de valores nulos em cada coluna da tabela
+# colunas com tipagem incorreta: 'Spotify Streams', 'Spotify Playlist Count', 'Spotify Playlist Reach', 'YouTube Views', 'YouTube Likes', 'TikTok Posts','TikTok Likes', 'TikTok Views', 'YouTube Playlist Reach','AirPlay Spins', 'SiriusXM Spins', 'Deezer Playlist Reach', 'Pandora Streams', 'Pandora Track Stations', 'Soundcloud Streams', 'Shazam Counts'],
+
+colunas_tipagem = ['Spotify Streams', 'Spotify Playlist Count', 'Spotify Playlist Reach', 'YouTube Views', 'YouTube Likes', 'TikTok Posts','TikTok Likes', 'TikTok Views', 'YouTube Playlist Reach','AirPlay Spins', 'SiriusXM Spins', 'Deezer Playlist Reach', 'Pandora Streams', 'Pandora Track Stations', 'Soundcloud Streams', 'Shazam Counts']
+df_mais_ouvidas[colunas_tipagem] = df_mais_ouvidas[colunas_tipagem].apply(lambda x: x.str.replace(',', '').astype('float'))
+print(df_mais_ouvidas[colunas_tipagem])
+
+# ------------------------------------------------PASSO 3------------------------------------------------
+# Corrija a coluna 'Release Date' para o formato datetime.
+#--------------------------------------------------------------------------------------------------------
+
+# print(df_mais_ouvidas.dtypes)
+df_mais_ouvidas['Release Date'] = pd.to_datetime(df_mais_ouvidas['Release Date'], format='mixed')
```

Comment:
também é possível fazer com:
```suggestion
df_mais_ouvidas['Release Date'] = pd.to_datetime(df_mais_ouvidas['Release Date'], format='%m/%d/%YYYY')
```